components/TARGET_PSA/services/crypto/psa_psa_f_ifs.h

```diff
@@ -0,0 +1,39 @@
+/* Copyright (c) 2017-2018 ARM Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/***********************************************************************************************************************
+ * !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+ * THIS FILE IS AN AUTO-GENERATED FILE - DO NOT MODIFY IT.
```

Comment:
@gilles-peskine-arm we'll add an explanation. but lack of explanation is not classified as a critical issue

Reply:
@gilles-peskine-arm,
this is a great input! Fixed https://github.com/ARMmbed/mbed-os/pull/8744/commits/b6c86e63d04bd92f1cbbd90ba3a96bf5b65e5b55 here.
once rebased - execute `$python tools/spm/generate_partition_code.py` to regenerate these files

Reply:
@avolinski It's a critical issue because this code is going into production and we need to know how to re-generate it. We can't add the explanation after shipping because then there would be no way to go from the shipped version to the explanation. @alzix Thanks!

Reply:
`tools/spm/generate_partition_code.py` doesn't take parameters and runs on the whole repository, right? Then everything is set to keep track of this information, and this issue is resolved.